craft_application/services/provider.py

```diff
@@ -340,9 +340,7 @@ def _capture_logs_from_instance(self, instance: craft_providers.Executor) -> Non
         ) as log_path:
             if log_path:
                 emit.debug("Logs retrieved from managed instance:")
-                with log_path.open() as log_file:
-                    for line in log_file:
-                        emit.debug(":: " + line.rstrip())
+                emit.dump_log_contents(log_path)
```

Comment:
Blocking on [this discussion](https://github.com/canonical/craft-cli/pull/331#discussion_r2010637639)

Reply:
Discussion complete, this needs to be changed :-)